sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -179,16 +197,33 @@ def __iter__(self):
         #   May need to revise.
         trigger_driver = trigger.create_trigger_driver(self._windowing, True)
         windowed_key_values = trigger_driver.process_entire_key
-      coder_impl = self._post_grouped_coder.get_impl()
+
       key_coder_impl = self._key_coder.get_impl()
+      coder_impl = self._post_grouped_coder.get_impl()
       for encoded_key, windowed_values in self._table.items():
         key = key_coder_impl.decode(encoded_key)
+        output_stream = create_OutputStream()
         for wkvs in windowed_key_values(key, windowed_values):
           coder_impl.encode_to_stream(wkvs, output_stream, True)
-      self._grouped_output = [output_stream.get()]
+
+        self._grouped_output.append(output_stream.get())
       self._table = None
+
+  def __iter__(self):
```

Comment:
Do we actually use raw `__iter__` anywhere anymore? If not, remove. If so, perhaps implement as iter(self.partition(1)).